_ruby_libs/text_rendering.rb

```diff
@@ -12,17 +13,34 @@ def rst_to_md(rst)
   end
 end
 
-# Modifies markdown image links so that they link to github user content
-def fix_image_links(text, raw_uri, additional_path = '')
+# Modifies markdown local links so that they link to github user content
+def fix_local_links(text, raw_uri, browse_uri)
   readme_doc = Nokogiri::HTML(text)
-  readme_doc.xpath("//img[@src]").each() do |el|
-    #puts 'img: '+el['src'].to_s
-    unless el['src'].start_with?('http')
-      el['src'] = ('%s/%s/' % [raw_uri, additional_path])+el['src']
+  readme_doc.css("img[src]").each() do |img|
```

Comment:
nitpick: parentheses in Ruby when not required are usually omitted, especially for methods that take no arguments.

If there is a separate styleguide that mandates parentheses here that overrides this nitpick.

Reply:
just removed the unnecessary parentheses, thanks!